doc/Pacemaker_Explained/en-US/Ch-Advanced-Resources.txt

```diff
@@ -427,6 +422,10 @@ Thus in order to indicate that +clone:0+ will be started on +sles-1+,
 +clone:2+ will be started on +sles-3+, and +clone:3+ will be started
 on +sles-2+, the cluster would set
 
++OCF_RESKEY_CRM_meta_notify_inactive_resource+ is an exception as the
+matching +uname+ variable deosn't exists since inactive resources
+are not running on any node.
+
```

Comment:
Looks good, but this paragraph needs to move above the previous one, and "deosn't" -> "doesn't".